Assets/Scripts/DialogueSystem/DialogueManager.cs

```diff
@@ -93,6 +93,18 @@ public void ReadNext()
                 Invoke("OpenMemoryMatch", 2.5f);
                 letsGoAnimator.SetBool("OpenMM", true);
             }
+
+            if (speakerName.text == "Minion")
```

Comment:
this if and the ones content above it are so similar. I feel like it could be reduced within another function for these cases.